cmds-main: Add intel plugin commands

Arunpandian15 opened the pull request:
Added intel plugin commands in list

Signed-off-by: Arunpandian J <apj.arun@samsung.com>

igaw commented:
What about adding in nvme.txt a new section called 'Plugins/Vendor extension commands' and do a 'include cmd-plugins.txt' and have these intel commands added in this new file?

Arunpandian15 commented:
Thanks @igaw 
I have added intel plugin commands in new file cmd-plugins.txt and created new section Plugins/Vendor extension commands, added include cmd-plugins.txt there.

igaw commented:
Thanks!
